### ISSUE TITLE
Crash Using Alamofire Within App Clip

### ISSUE THREAD

**Issue by AssyriaGameStudio** [Alamofire/Alamofire#3255]
[ * ] I've read, understood, and done my best to follow the [*CONTRIBUTING guidelines](https://github.com/Alamofire/Alamofire/blob/master/CONTRIBUTING.md).

## What did you do?
Added the Alamofire to my App Clip target using Cocapods

## What did you expect to happen?

It to work as per other targets.

## What happened instead?

I get an on-launch crash with the error:

```
dyld: Library not loaded: @rpath/Alamofire.framework/Alamofire
  Referenced from: /private/var/containers/Bundle/Application/53C78E93-79E0-438F-AA1C-11324FE1B0D4/SleepiestAppClip.app/SleepiestAppClip
  Reason: image not found
dyld: launch, loading dependent libraries
DYLD_LIBRARY_PATH=/usr/lib/system/introspection
DYLD_INSERT_LIBRARIES=/Developer/usr/lib/libBacktraceRecording.dylib:/Developer/usr/lib/libMainThreadChecker.dylib:/Developer/Library/PrivateFrameworks/DTDDISupport.framework/libViewDebuggerSupport.dylib
(lldb) 
```

Pod File:
```
use_frameworks!

target 'Sleepiest' do
platform :ios, '11.0'
pod 'Alamofire', '~> 5.2'
end

target 'SleepiestAppClip' do
platform :ios, '14.0'
pod 'Alamofire', '~> 5.2'
end
```

## Alamofire Environment

**Alamofire version: 5.2
**Xcode version: 12.0 beta 2
**Swift version: 5
**Platform(s) running Alamofire: iOS App Clip
**macOS version running Xcode: 10.15.4

## Demo Project

Seems to happen with any empty project with an App Clip target.

**Comment by jshier** [comment]
This is an issue with CocoaPods which is fixed on their `master` branch. In the mean time you could try their prerelease version or use SPM through Xcode to integrate your dependencies.

**Comment by jshier** [comment]
CocoaPods has released their first 1.10 beta, which officially supports AppClips.
